CMakeLists.txt

```diff
@@ -47,6 +47,11 @@ set(CPACK_DEBIAN_PACKAGE_MAINTAINER "Target Devel <target-devel@vger.kernel.org>
 set(CPACK_DEBIAN_PACKAGE_HOMEPAGE "https://github.com/open-iscsi/tcmu-runner")
 include(CPack)
 
```

Comment:
I think you need to rebase and repush by doing a forced push, so we do not end up seeing this in the final pull and commit.

If everyone adds junk like this then it gets messy in the main git tree.